articles/preparing_for_consoles.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Preparing for consoles
+description: How to get your game ready to run on consoles.
+---
+
+# Preparing for consoles
+
+If you would like to port your game to consoles, there are some best practices to follow if you want to avoid running into issues while porting.
+
+MonoGame for gaming consoles uses a dedicated .NET runtime that is not maintained by Microsoft. Therefore, a 100% accuracy and reliability is not guaranteed. Moreover, the console runtime makes use of `ahead-of-time` native compilation (AOT), which means that some .NET features will not, and cannot work on consoles.
+
+This article explains the most common pitfalls and suggested guidelines to optimize your chances of having a smoother porting experience.
+
+> [!NOTE]
+> *Gaming consoles are restricted to registered developers and are not publicly available nor publicly documented. To get access to those platforms, please contact your console account manager(s). MonoGame documentation for closed platforms is available in their respective repositories.*
+
+## No use of runtime reflection
+
+The main show stopper when it comes to porting a .NET game to consoles, is the use of runtime [reflection](https://learn.microsoft.com/en-us/dotnet/fundamentals/reflection/reflection).
+
+Reflection is, at large, unsupported in an AOT context. Which means that anywhere you or a third party library uses reflection, your game will crash.
+
+In order to make sure that your game abides to that rule, you can try to publish an AOT'd version of it for desktop computers and verify how it fares.
+
+To publish with AOT:
+
+- Add `<PublishAot>true</PublishAot>` to your `.csproj`.
+- Then run `dotnet publish` from the command-line/terminal.
+
+This will nativily compile your game in a fashion similar to consoles. The output executable will be in your output folder, within a sub-folder nammed `publish`.
+
+From there, you can try running this executable. If it does not start or crashes later on, you likely are using reflection or another unsupported feature in a AOT runtime.
+
+Native executables can debugged by:
+
+- Atarting an empty Visual Studio with no code.
```

Comment:
Typo, I assume it is supposed to be 'Starting'

Reply:
Fixed